actionpack/lib/action_dispatch/middleware/templates/rescues/diagnostics.html.erb

```diff
@@ -8,11 +8,8 @@
 </header>
 
 <div id="container">
-  <h2>
-    <%= h @exception.message %>
-
-    <%= render "rescues/actions", exception: @exception, request: @request %>
-  </h2>
+  <%= render "rescues/message_with_did_you_mean", exception: @exception %>
```

Comment:
Running all exceptions through a template named like this feels... weird. Especially as it then contains a big conditional to handle the (actually very common) case that this is not 'with did_you_mean' at all. 🤔

Reply:
I've renamed the template to `message_and_suggestions`.
There are also more exception templates that could use the partial but I'm hesitant to add it to templates for exceptions that don't support DidYouMean yet.